templates/repo/issue/view_content/pull.tmpl

```diff
@@ -322,176 +322,62 @@
-										</button>
-										{{if .IsPullBranchDeletable}}
-											<div class="ui checkbox ml-2">
-												<input name="delete_branch_after_merge" type="checkbox" {{if $prUnit.PullRequestsConfig.DefaultDeleteBranchAfterMerge}}checked{{end}}>
-												<label>{{$.i18n.Tr "repo.branch.delete" .HeadTarget}}</label>
-											</div>
-										{{end}}
-									</form>
-								</div>
-							{{end}}
-							<div class="dib">
-								<div class="ui {{if $notAllOverridableChecksOk}}red{{else}}green{{end}} buttons merge-button">
-									<button class="ui button" data-do="{{.MergeStyle}}">
-										{{svg "octicon-git-merge"}}
-										<span class="button-text">
-										{{if eq .MergeStyle "merge"}}
-											{{$.i18n.Tr "repo.pulls.merge_pull_request"}}
-										{{end}}
-										{{if eq .MergeStyle "rebase"}}
-											{{$.i18n.Tr "repo.pulls.rebase_merge_pull_request"}}
-										{{end}}
-										{{if eq .MergeStyle "rebase-merge"}}
-											{{$.i18n.Tr "repo.pulls.rebase_merge_commit_pull_request"}}
-										{{end}}
-										{{if eq .MergeStyle "squash"}}
-											{{$.i18n.Tr "repo.pulls.squash_merge_pull_request"}}
-										{{end}}
-										{{if eq .MergeStyle "manually-merged"}}
-											{{$.i18n.Tr "repo.pulls.merge_manually"}}
-										{{end}}
-										</span>
-									</button>
-									{{if gt $prUnit.PullRequestsConfig.AllowedMergeStyleCount 1}}
-										<div class="ui dropdown icon button no-text">
-											{{svg "octicon-triangle-down" 14 "dropdown icon"}}
-											<div class="menu">
-												{{if $prUnit.PullRequestsConfig.AllowMerge}}
-												<div class="item{{if eq .MergeStyle "merge"}} active selected{{end}}" data-do="merge">{{$.i18n.Tr "repo.pulls.merge_pull_request"}}</div>
-												{{end}}
-												{{if $prUnit.PullRequestsConfig.AllowRebase}}
-												<div class="item{{if eq .MergeStyle "rebase"}} active selected{{end}}" data-do="rebase">{{$.i18n.Tr "repo.pulls.rebase_merge_pull_request"}}</div>
-												{{end}}
-												{{if $prUnit.PullRequestsConfig.AllowRebaseMerge}}
-												<div class="item{{if eq .MergeStyle "rebase-merge"}} active selected{{end}}" data-do="rebase-merge">{{$.i18n.Tr "repo.pulls.rebase_merge_commit_pull_request"}}</div>
-												{{end}}
-												{{if $prUnit.PullRequestsConfig.AllowSquash}}
-												<div class="item{{if eq .MergeStyle "squash"}} active selected{{end}}" data-do="squash">{{$.i18n.Tr "repo.pulls.squash_merge_pull_request"}}</div>
-												{{end}}
-												{{if and $prUnit.PullRequestsConfig.AllowManualMerge $.IsRepoAdmin}}
-												<div class="item{{if eq .MergeStyle "manually-merged"}} active selected{{end}}" data-do="manually-merged">{{$.i18n.Tr "repo.pulls.merge_manually"}}</div>
-												{{end}}
-											</div>
-										</div>
-									{{end}}
-								</div>
-							</div>
+
+							<script>
+								<!-- /* eslint-disable */ -->
```

Comment:
For further readers: at the moment, "eslint-disable" + "script(type=module)" + go-template can not work together.